### ISSUE TITLE
Feature Request: alt text for bgset plugin

### ISSUE THREAD

**Issue by MikeMcChillin** [aFarkas/lazysizes#521]
https://github.com/aFarkas/lazysizes/blob/gh-pages/plugins/bgset/ls.bgset.js#L145

Would be great to check for a `data-alt="My Alt Text"` or similar on the element and add that text instead of defaulting to empty.

**Comment by aFarkas** [comment]
The bgset plugin is the plugin that you need if you want responsive and lazyloading for **background images**. If you want an image with an `alt` attribute. Than use the `img` element directly there is no one stopping you.

**Comment by rickydazla** [comment]
@aFarkas I think the issue @MikeMcChillin was hoping to solve is that this plugin generates a (hidden) picture element and since it is missing an alt attribute there is some penalization in tools that are measuring accessibility (maybe I'm wrong on all of that, but that's why I'm here anyway 😄)

**Comment by aFarkas** [comment]
To be honest I should add a `role="presentation"` to make clear that this is a pure presentational and no tool should punish you for that.

My question: What is your use case? Why do you use bgset instead of the `picture` element?

**Comment by rickydazla** [comment]
Yes @aFarkas that would be better indeed! I'm setting a background image on a modal overlay, the image is purely ornamental..

**Comment by paavo** [comment]
Is there a way to define the ALT-Tag used for the placeholder <img> Tag here @aFarkas ?
https://github.com/aFarkas/lazysizes/blob/499f0f834f8421d0a1d4754c1863f37eb8ebfe36/plugins/bgset/ls.bgset.js#L158

Noone likes empty alt Tags for Images in the DOM - even if they are hidden.

**Comment by mikesnoeren** [comment]
For anyone still needing a fix/workaround for this: I wrote a simple script that checks for `data-bgset-alt` and adds it to the images within the `data-bgset` element.

```js
const lazySizesAlt = document.querySelectorAll('[data-bgset][data-bgset-alt]');

lazySizesAlt.forEach((e) => {
  const img = e.querySelector('.lazyautosizes');
  const alt = e.getAttribute('data-bgset-alt');
  img.setAttribute('alt', alt);
});
```

**Comment by ccamaralbr** [comment]
hi there @aFarkas , any updates on the above context? thx!

**Comment by afwebmaster** [comment]
I found a temporal solution for those using this library for background images:
I achieved this by editing the lazysizes.min.js (I don't have access to the unminified library).

1 - Search for 'alt'. You should get only 1 result, where alt is set to ""
2 - Replace the "" with 
`t.target.getAttribute([ALT_ATTRIBUTE])!==undefined&&t.target.getAttribute([ALT_ATTRIBUTE])!==''?t.target.getAttribute([ALT_ATTRIBUTE]):''`

Note that [ALT_ATTRIBUTE] has to be replaced with the name of the attribute from the node that is getting the image info from. In my case it is "aria-label".

I hope it works for you guys :)
